src/a-st-ext.adoc

```diff
@@ -18,6 +18,10 @@ appear to have finally settled on release consistency as the standard
 memory consistency model and so the RISC-V atomic support is built
 around this model.
 ====
+
+The A standard extension comprises instructions provided by the Zaamo and Zalrsc
```

Comment:
I believe we can remove the word standard here, as well as from the section headings in order to be consistent with the rest of the spec.

Reply:
Updated.